.github/workflows/stale.yml

```
@@ -0,0 +1,57 @@
+name: 'Close stale issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+env:
+  LC_ALL: en_US.UTF-8
+
+defaults:
+  run:
+    shell: bash
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
```

Same argument as my first inline comment

```suggestion
  close-issues-and-pull-requests:
```